### PR TITLE
feat: prisma initial config address table

### PR DESCRIPTION
**Pull request by bol507**
## Description

create prisma initial configuration

## Summary of changes

add two tables address and address_history for store gps address

## Checklist

- [x] Correctly use conventional commits
- [x] Update documentation
- [x] Just modify what the branch mentions
- [x] My code works correctly

## Notes
to use prisma studio run this command
npx prisma studio

every change in prisma please generate  in project.  run this command
npx prisma generate

### CODE REVIEW

File: prisma/schema.prisma
Comment (Heba-WebDev) [inline prisma/schema.prisma:1]: Por que tiene url la tabla de address?

Comment (bol507) [reply prisma/schema.prisma:1]: The address table is to save the GPS address, I placed it to test if Prisma was working.

Comment (jom3) [reply prisma/schema.prisma:1]: both tables have a default value at updated_at field, it must be null or empty at the beginning

Comment (bol507) [reply prisma/schema.prisma:1]: in this example created_at DateTime @default(now())
  updated_at DateTime @default(now())   the function now generate date in server .

**Pull request merged**